sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-batch-deployment.ipynb

```diff
@@ -0,0 +1,693 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Azure AI Content Safety (AACS) enabled text-to-image batch endpoint (Preview)\n",
+    "### This notebook will walk you through the steps to create an __Azure AI Content Safety__ enabled __text-to-image__ batch endpoint.\n",
+    "### This notebook is under preview\n",
+    "### The steps are:\n",
+    "1. Create an __Azure AI Content Safety__ resource for moderating the request from user and response from the __text-to-image__ batch endpoint.\n",
+    "2. Create a new __Azure AI Content Safety__ enabled __text-to-image__ batch endpoint with a custom score.py which will integrate with the __Azure AI Content Safety__ resource to moderate the response from the __text-to-image__ model and the request from the user. To make the custom score.py to successfully authenticated to the __Azure AI Content Safety__ resource, use __Environment variable__ to pass the access key of the __Azure AI Content Safety__ resource to the custom score.py via environment variable. The custom score.py can use the key directly to access the Azure AI Content Safety resource, this option is less secure, if someone in your org has access to the endpoint, he/she can get the access key from the environment variable and use it to access the Azure AI Content Safety resource.\n",
```

Comment:
1) Can we break this in to 2 sentences as below:

> Create a new Azure AI Content Safety enabled text-to-image batch endpoint with a custom score.py script. This script will integrate with the AACS resource to moderate the response from the text-to-image model and the request from the user. 

2) We can define acronym for Azure AI Content Safety (AACS) once and then use acronym every else.

Reducing sentence size will make it more understandable. This just my perception; happy to discuss if needed.

Reply:
Thanks. Addressed it,